test/src/main/java/org/apache/accumulo/test/shell/ShellIT.java

```diff
@@ -614,4 +615,15 @@ public void testMaxSplitsOption() throws Exception {
         "0\n1\n2\n3\n4\n5\n6\n7\n8\n9\na\nb\nc\nd\ne\nf\ng\nh\ni\nj\nk\nl\nm\nn\no\np\nq\nr\ns\nt\n");
   }
 
+  @Test
+  public void testGetTimeType() throws Exception {
+    Shell.log.debug("Starting testGetTimeType test -----------------");
+    exec("createtable tmtype", true);
+    exec("gettimetype", true, TimeType.MILLIS.toString());
+    exec("gettimetype -t tmtype", true, TimeType.MILLIS.toString());
```

Comment:
Could also try a user table with logical time to make the test cover a bit more.

```suggestion
    exec("gettimetype -t tmtype", true, TimeType.MILLIS.toString());
    exec("createtable -tl logicaltt", true);
     exec("gettimetype -t logicaltt", true, TimeType.LOGICAL.toString());
```